components/blitz/resources/omero/FS.ice

```diff
@@ -398,6 +399,7 @@ module omero {
                                 int blockSize,
                                 bool ignoreSysFiles,
                                 bool ignoreDirEvents,
+                                bool platformCheck,
```

Comment:
Yes, I realised I had changed the API here but wondered how much impact it has given its isolation to the DropBox system. Can a (final) parameter be optional in Ice? There is probably an alternative but it may not be straightforward.